### ISSUE TITLE
isLoading from redux bug

### ISSUE THREAD

**Issue by maystrenkoYurii** [faceyspacey/react-universal-component#181]
isLoading from redux bug causes looping of the render component
![default](https://user-images.githubusercontent.com/23016311/53700489-8a6d1100-3dfb-11e9-92d9-a24f0df7aa63.PNG)

**Comment by ScriptedAlchemy** [comment]
Any other debugging info?

**Comment by maystrenkoYurii** [comment]
Yes. Clone this project https://gitlab.com/yura1994hd/middle/tree/bug_react-universal-coponent. Checout on bug_react-universal-coponent and npm run start:dev:local. After run uncomment code \src\containers\Routes\index.js
![default](https://user-images.githubusercontent.com/23016311/53726183-25a2cc80-3e76-11e9-8419-86625e9633fa.PNG)

**Comment by maystrenkoYurii** [comment]
It seems I found the reason. What seems to me to be the fault of the react-router

Good work:
`<Route exact path={constants.PATCH_URL_BASKET} component={Basket} />`
Looping:
`<Route
      exact
      path={constants.PATCH_URL_BASKET}
      component={props => <Basket {...props} />}
 />`
